Message/AndroidMessage.php

```diff
@@ -136,7 +150,9 @@ public function setDeviceIdentifier($identifier)
      */
     public function getTargetOS()
     {
-        return ($this->isGCM ? Types::OS_ANDROID_GCM : Types::OS_ANDROID_C2DM);
+        if($this->isGCM) return Types::OS_ANDROID_GCM;
```

Comment:
Please reformat this as per PSR-2 if possible - makes it a bit easier to read :-)